src/engine/resource_manager.cpp

```diff
@@ -69,6 +85,66 @@ Tileset *ResourceManager::lookup_tileset(TilesetHandle handle) {
   return &r_it->second;
 }
 
+FontHandle ResourceManager::load_font(const char *path) {
+  std::vector<char *> lines = load_file_by_lines(path);
+  Font f;
+  u32 line_height = 0;
+  u32 base = 0;
+  u32 w = 0;
+  u32 h = 0;
+  const char* font_tex_folder = "assets/fonts/";
+  const u32 FONT_FOLDER_PATH_LEN=13;
+  char tex_path[1024+FONT_FOLDER_PATH_LEN];
```

Comment:
This string loading is a bit dodgy & open to buffer overflows, should add a TODO or something at least (+ add to trello)